### PR TITLE
adding new admin ui to input xacml policies

### PR DESCRIPTION
**Pull request by lahiru823**

### CODE REVIEW

File: components/appmgt/org.wso2.carbon.appmgt.api/src/main/java/org/wso2/carbon/appmgt/api/APIProvider.java [hunk 1]
```diff
@@ -178,11 +178,12 @@ public void generateEntitlementPolicies(APIIdentifier apiIdentifier) throws
      * @param policyPartial     policy content
      * @param isSharedPartial   policy status
      * @param policyAuthor      author of the policy
+     * @param policyPartialDesc policy description
      * @return policy id
      * @throws AppManagementException
      */
     int saveEntitlementPolicyPartial(String policyPartialName, String policyPartial, boolean isSharedPartial,
-                                     String policyAuthor) throws AppManagementException;
+                                     String policyAuthor,String policyPartialDesc) throws AppManagementException;
```
Comment (dinusha-dilrukshi) [inline components/appmgt/org.wso2.carbon.appmgt.api/src/main/java/org/wso2/carbon/appmgt/api/APIProvider.java:186]: Add space after,

File: components/appmgt/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/APIProviderImpl.java [hunk 2]
```diff
@@ -468,15 +468,15 @@ public int getWebAppId(String uuid) throws AppManagementException {
 
     @Override
     public int saveEntitlementPolicyPartial(String policyPartialName, String policyPartial, boolean isSharedPartial,
-                                            String policyAuthor) throws AppManagementException {
-        return appMDAO.saveEntitlementPolicyPartial(policyPartialName, policyPartial, isSharedPartial, policyAuthor);
+                                            String policyAuthor,String policyPartialDesc) throws AppManagementException {
```
Comment (dinusha-dilrukshi) [inline components/appmgt/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/APIProviderImpl.java:471]: Add space between params.

File: components/appmgt/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/dao/AppMDAO.java [hunk 3]
```diff
@@ -5371,6 +5375,7 @@ public int getWebAppId(String uuid) throws AppManagementException {
                 policyPartial.setPolicyPartialContent(rs.getString("CONTENT"));
                 policyPartial.setShared(rs.getBoolean("SHARED"));
                 policyPartial.setAuthor(rs.getString("AUTHOR"));
+				policyPartial.setDescription(rs.getString("DESCRIPTION"));
```
Comment (dinusha-dilrukshi) [inline components/appmgt/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/dao/AppMDAO.java:5378]: Formatting issue.

**Comment by lahiru823** [comment]
fixed all mentioned formatting issues with commit # https://github.com/wso2/carbon-appmgt/commit/376e6f4d59718ec892abc77b3683f0fca2ea7fa0

File: components/appmgt/org.wso2.carbon.appmgt.api/src/main/java/org/wso2/carbon/appmgt/api/APIProvider.java (hunk 1 above)
Comment (lahiru823) [inline components/appmgt/org.wso2.carbon.appmgt.api/src/main/java/org/wso2/carbon/appmgt/api/APIProvider.java:186]: fixed formatting issues with commit # https://github.com/wso2/carbon-appmgt/commit/376e6f4d59718ec892abc77b3683f0fca2ea7fa0

File: components/appmgt/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/APIProviderImpl.java (hunk 2 above)
Comment (lahiru823) [inline components/appmgt/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/APIProviderImpl.java:471]: fixed formatting issues with commit # https://github.com/wso2/carbon-appmgt/commit/376e6f4d59718ec892abc77b3683f0fca2ea7fa0

File: components/appmgt/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/dao/AppMDAO.java (hunk 3 above)
Comment (lahiru823) [inline components/appmgt/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/dao/AppMDAO.java:5378]: fixed formatting issues with commit # https://github.com/wso2/carbon-appmgt/commit/376e6f4d59718ec892abc77b3683f0fca2ea7fa0

**Pull request merged**